homeassistant/components/notify/html5.py

```diff
@@ -120,6 +126,12 @@ def get_service(hass, config, discovery_info=None):
     if registrations is None:
         return None
 
+    vapid_pub_key = config.get(ATTR_VAPID_PUB_KEY)
+    vapid_prv_key = config.get(ATTR_VAPID_PRV_KEY)
+    vapid_email = config.get(ATTR_VAPID_EMAIL)
+
+    hass.http.register_view(HTML5ApplicationServerKeyView(vapid_pub_key))
```

Comment:
Do we need null guard for `vapid_pub_key` since it is an optional config
```python
if vapid_pub_key is not None:
    hass.http.register_view(HTML5ApplicationServerKeyView(vapid_pub_key))
```